app/qml/components/MMCodeScanner.qml

```diff
@@ -54,27 +54,16 @@ Drawer {
     }
   }
 
-  Canvas {
-    id: canvas
-
-    anchors.fill: parent
-    opacity: 0.8
-
-    onPaint: {
-      var ctx = getContext("2d");
-      let w = parent.width
-      let h = parent.height
-      ctx.fillStyle = __style.nightColor
-      ctx.fillRect(0, 0, parent.width, parent.height);
-      ctx.fill();
-      if(parent.width < parent.height)
-        ctx.clearRect(w / 4, h / 2 - w / 4, w / 2, w / 2)
-      else
-        ctx.clearRect(w / 2 - h / 4, h / 4, h / 2, h / 2)
-    }
+  function unload() {
+    qrcodeScanner.videoSink = null
+    camera.active = false
+    captureSession.videoOutput = null
+    captureSession.camera = null
   }
 
   Item {
```

Comment:
Use MMTextBubble instead of all this Item {} code

Reply:
This blue "I" image needs to be probably exported and added to images/ not sure we have it already?